tardis/visualization/tools/sdec_plot.py

```diff
@@ -1250,7 +1254,7 @@ def _plot_emission_mpl(self):
             except:
                 # Add notifications that this species was not in the emission df
                 if self._species_list is None:
-                    longer.info(
+                    logger.info(
```

Comment:
Good catch, don't know how this got committed - can you please do these changes (longer -> logger) in a separate PR instead? 

~ One PR should work on one idea so that code history remains relevant :)

Reply:
Done, I have dropped the commit for the same ... I will wait for this PR to get merged ... then I will send in another one fixing this ^^